guides/v2.1/install-gde/composer.md

```diff
@@ -0,0 +1,102 @@
+-   Create the [Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html)
+-   [Install Composer](https://getcomposer.org/download/){:target="\_blank"}
+-   Obtain [authentication keys]({{page.baseurl}}/install-gde/prereq/connect-auth.html) for the Magento code repository
+
+## Get the metapackage
+
+To get the Magento metapackage:
+
+1.  Log in to your Magento server as, or switch to, the [Magento file system owner]({{page.baseurl}}/install-gde/prereq/file-sys-perms-over.html).
+1.  Change to the web server docroot directory or a directory that you have configured as a virtual host docroot.
+1.  Create a new Composer project using the {{site.data.var.ce}} or {{site.data.var.ee}} Magento metapackage.
+
+    **{{site.data.var.ce}}**
+    ```bash
+    composer create-project --repository=https://repo.magento.com/ magento/project-community-edition <install-directory-name>
+    ```
+
+    **{{site.data.var.ee}}**
+    ```bash
+    composer create-project --repository=https://repo.magento.com/ magento/project-enterprise-edition <install-directory-name>
+    ```
+
+    When prompted, enter your Magento authentication keys. Your _public key_ is your username; your _private key_ is your password.
+
+    If you encounter errors, such as `Could not find package...` or `...no matching package found`, make sure there are no typos in your command. If you still encounter errors, you may not be authorized to download {{site.data.var.ee}}. Contact [Magento support](https://magento.com/support) for help.
+
+    See [troubleshooting]({{page.baseurl}}/install-gde/trouble/tshoot_composer-fail.html) for help with more errors.
+
+## Set file permissions
+
+You must set read-write permissions for the web server group before you install the Magento software. This is necessary so that the Setup Wizard and command line can write files to the Magento file system.
+
+```terminal
+cd /var/www/html/<magento install directory>
+find var generated vendor pub/static pub/media app/etc -type f -exec chmod g+w {} \;
+find var generated vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} \;
+chown -R :www-data . // Ubuntu
+chmod u+x bin/magento
+```
+
+## Install Magento
+
+There are two options for installing Magento:
+
+-  Command line
+-  Web Setup Wizard
+
+### Command line
+{:.no_toc}
+
+This example assumes that the Magento install directory is named `magento2ee`, the `db-host` is on the same machine (`localhost`), and that the `db-name`, `db-user`, and `db-password` are all `magento`:
+
+```bash
+bin/magento setup:install --base-url=http://localhost/magento2ee --db-host=localhost --db-name=magento --db-user=magento --db-password=magento --backend-frontname=admin --admin-firstname=admin --admin-lastname=admin --admin-email=admin@admin.com --admin-user=admin --admin-password=admin123 --language=en_US --currency=USD --timezone=America/Chicago --use-rewrites=1
+```
```

Comment:
Format feedback: Not sure if possible, but would be great for this command to wrap on the generated page and still be copied and used. Readers can use a horizontal scrolled.